### PR TITLE
three seconds is better test1

### PR DESCRIPTION
**Pull request by yulyfish**
three seconds is better test1 of comment

### CODE REVIEW

**Comment by yuly** [comment]
oh,I see you modify,please continue optimize

**Comment by yulyfish** [comment]
recevie, i check it

File: blink.ino [hunk 1]
```diff
@@ -18,7 +18,7 @@ void setup() {
 // the loop routine runs over and over again forever:
 void loop() {
   digitalWrite(led, HIGH);   // turn the LED on (HIGH is the voltage level)
-  delay(1000);               // wait for a second
+  delay(3001);               // wait for a second
```
Comment (yulyfish) [inline blink.ino]: 3000->3001

Comment (yulyfish) [reply blink.ino]: 11

Comment (yulyfish) [reply blink.ino]: 222

**Review by yulyfish — COMMENTED** [review]
3333

File: blink.ino (hunk 1 above)
Comment (yuly) [reply blink.ino]: yuly6666

Comment (yulyfish) [reply blink.ino]: fish777

**Comment by yulyfish** [comment]
fish888

**Pull request merged**